src/libsync/capabilities.cpp

```diff
@@ -103,6 +103,20 @@ int Capabilities::defaultPermissions() const
     return _fileSharingCapabilities.value(QStringLiteral("default_permissions"), 1).toInt();
 }
 
+std::chrono::seconds Capabilities::remotePollInterval() const
+{
+    // The default of the capability is 60, but the clients use 30.
+    // Values below 5000 are not allowed and fall back to the default.
+    std::chrono::seconds interval(0);
```

Comment:
Where do we check for the 0 seconds interval?

Reply:
We don't explicitely but it is implicitely done by checking for `< seconds(5)`.

Reply:
Ok I mean in that case you can skip the check for 4999 and ok here and just return std::milliseconds(tmpInt)